cpp/daal/src/algorithms/kmeans/kmeans_lloyd_batch_impl.i

```diff
@@ -174,37 +175,49 @@ Status KMeansBatchKernel<method, algorithmFPType, cpu>::compute(const NumericTab
 
             for (size_t i = 0; i < nClusters; i++)
             {
-                if (clusterS0[i] > 0)
-                {
-                    const algorithmFPType coeff = 1.0 / clusterS0[i];
+                if (clusterS0[i] == 0) {
```

Comment:
`{` should be on separate row. Use `clang-format -i {files}` to apply code style:
conda install -c conda-forge clang-tools

Reply:
Fixed